examples/src/main/java/com/example/ftclibexamples/InterpLUTSample.java

```diff
@@ -36,14 +35,22 @@ public void runOpMode() throws InterruptedException
         //generating final equation
         lut.createLUT();
 
-        leftEncoder = new JSTEncoder(hardwareMap, "left");
-        rightEncoder = new JSTEncoder(hardwareMap, "right");
-        perpEncoder = new JSTEncoder(hardwareMap, "perp");
+        leftEncoder = new MotorEx(hardwareMap, "left");
+        rightEncoder = new MotorEx(hardwareMap, "right");
+        perpEncoder = new MotorEx(hardwareMap, "perp");
 
         // REVcoders
-        leftEncoder.setDistancePerPulse(2 / (double)8192);
-        rightEncoder.setDistancePerPulse(2 / (double)8192);
-        perpEncoder.setDistancePerPulse(2 / (double)8192);
+        leftEncoder.setDistancePerPulse(2 / (double) 8092);
+        rightEncoder.setDistancePerPulse(2 / (double) 8092);
+        perpEncoder.setDistancePerPulse(2 / (double) 8092);
+
+        odometry = new HolonomicOdometry(
+                leftEncoder::getDistance,
+                rightEncoder::getDistance,
+                perpEncoder::getDistance,
+                14,
```

Comment:
Should add comments for this and the following line, not self-explanatory, seems like magic numbers.